### PR TITLE
Delete the VMO from setting the default replicas to vmi-system-api

### PR DESCRIPTION
**Pull request by bhatpmk**
This change removes the code which sets the default replicas (which is 1) to vmi-system-api. This change is the first step to remove the dependency on verrazzano-monitoring-instance-api, further code clean-up will be handled separately.

### CODE REVIEW

File: pkg/vmo/vmospec.go [hunk 1]
```diff
@@ -85,9 +85,6 @@ func InitializeVMOSpec(controller *Controller, vmo *vmcontrollerv1.VerrazzanoMon
 	}
 
 	// Number of replicas for each component
-	if vmo.Spec.API.Replicas == 0 {
```
Comment (michael-cico) [inline pkg/vmo/vmospec.go:88]: I wonder if this change alone is good enough?  If we allow the replica count to default to 0, then maybe we don't have to set it on the VO side.

**Review by michael-cico — APPROVED** [review]

File: pkg/vmo/vmospec.go (hunk 1 above)
Comment (bhatpmk) [reply pkg/vmo/vmospec.go:88]: @michael-cico I have not tried that, will test that out.

Comment (bhatpmk) [reply pkg/vmo/vmospec.go:88]: @michael-cico I have confirmed that we don't need the change in Verrazzano Operator. Thank you.

File: pkg/vmo/vmospec.go [hunk 2]
```diff
@@ -85,9 +85,6 @@ func InitializeVMOSpec(controller *Controller, vmo *vmcontrollerv1.VerrazzanoMon
 	}
 
 	// Number of replicas for each component
```
Comment (pfmackin) [inline pkg/vmo/vmospec.go:87]: you need to add 2021 to the copyright

Comment (bhatpmk) [reply pkg/vmo/vmospec.go:87]: @pfmackin Thanks for noticing that, fixed now.

**Review by michael-cico — APPROVED** [review]

**Review by pfmackin — APPROVED** [review]

**Pull request merged**